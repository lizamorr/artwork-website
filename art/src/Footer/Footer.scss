.footer {
  position: absolute;
  padding: 10px 0;
  margin: 0;
  width: 100%;
  text-align: center;
  text-transform: uppercase;
  letter-spacing: 1px;
  font: normal 400 10px 'Montserrat', regular;
  color: #231f20;
  z-index: 1;
  margin-top: 94vh;
}
