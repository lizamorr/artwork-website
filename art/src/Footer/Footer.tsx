import React from 'react';

const Footer = () => {
  return <div className="footer">Â© Developed by Liza Morrison 2021.</div>;
};

export default Footer;
