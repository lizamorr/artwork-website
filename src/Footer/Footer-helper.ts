export const setFooterMargin = (isHome: boolean): void => {
  const footer = document.getElementById('footer');
  if (footer) {
    isHome
      ? (footer.style.marginTop = '94vh')
      : (footer.style.marginTop = '10vh');
  }
};
