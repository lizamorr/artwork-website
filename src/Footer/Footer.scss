.footer,
.footer-home {
  text-shadow: 1px 0px white;
  position: absolute;
  padding-bottom: 10px;
  width: 100%;
  max-width: 144rem;
  display: flex;
  justify-content: center;
  letter-spacing: 1px;
  font-variant: all-small-caps;
  font-size: 18px;
  @media screen and (min-width: 767px) {
    font-size: 1.5rem;
  }
  @media screen and (min-width: 1200px) {
    font-size: 2rem;
  }
  font-weight: bold;
  color: #231f20;
  z-index: 1;
}
.footer-home {
  margin-top: 80vh;
}
.footer {
  margin-top: 12vh;
}
