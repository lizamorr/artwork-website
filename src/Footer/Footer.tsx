import React from 'react';

const Footer: React.FC = () => {
  return <div className="footer">Â© Liza Morrison 2021.</div>;
};

export default Footer;
