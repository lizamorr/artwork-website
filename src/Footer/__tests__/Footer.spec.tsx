import Footer from '../Footer';
import renderer from 'react-test-renderer';
import React from 'react';

describe('Footer Component', () => {
  it('should render correctly', () => {
    const component = renderer.create(<Footer />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
