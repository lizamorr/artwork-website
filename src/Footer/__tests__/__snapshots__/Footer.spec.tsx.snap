// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component should render correctly 1`] = `
<div
  className="footer"
  id="footer"
>
  Â© Liza Morrison 2021
</div>
`;
